.azure-pipelines.yml

```diff
@@ -0,0 +1,45 @@
+trigger:
+- master
+
+pool:
+  vmImage: 'ubuntu-16.04'
+
+strategy:
+  matrix:
+    Node6:
+      NODE_VERSION: '6.x'
+      TEST_TITLE: 'Node 6 JUnit Test Results'
+    Node8:
+      NODE_VERSION: '8.x'
+      TEST_TITLE: 'Node 8 JUnit Test Results'
+    Node10:
+      NODE_VERSION: '10.x'
+      TEST_TITLE: 'Node 10 JUnit Test Results'
+
+steps:
+
+  - task: NodeTool@0
+    inputs:
+      versionSpec: $(NODE_VERSION)
+      checkLatest: true
```

Comment:
is `checkLatest` required? Alternatively, I wonder if you shouldn't lock to specific Node versions (e.g. 8.x -> `8.15.0`) instead, mainly to avoid a build failing because a new version of Node was released.